cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -859,7 +934,7 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
                     tabs[1].editors = [StaffLockEditor];
                 } else if (xblockInfo.isSequential()) {
                     tabs[0].editors = [ReleaseDateEditor, GradingEditor, DueDateEditor];
-                    tabs[1].editors = [ContentVisibilityEditor];
+                    tabs[1].editors = [ContentVisibilityEditor, ShowCorrectnessEditor];
```

Comment:
@sstack22 and @roderickmorales if we introduce a new Tab into the dialog, the code changes would be here (and minimal). Most of the impact would be to tests/doc.

My proposal is that we have a new Tab ("Visibility") for Subsection and Correctness Visibility, both of which are always shown. Then "Set as Special Exam" and "Prerequisite" sections appear on "Advanced", and the "Advanced" tab is only present if at least one of those features is enabled.